### ISSUE TITLE
Installation strategies

### ISSUE THREAD

**Issue by njgheorghita** [ethpm/ethpm-cli#31]
Make cli tool available for installation via both Pypi and Homebrew.

**Comment by cgewecke** [comment]
@njgheorghita Do you have any thoughts about also publishing a docker image? Solidity, Vyper and Geth all do this and it's made them pretty straightforward to consume in the JS build pipeline tools.

Happy to look into this and open a PR here if it's something you think worth doing...

**Comment by njgheorghita** [comment]
@cgewecke Yup, that's a great thought, thanks. A pr would be great, but you might want to wait until some kind of stable alpha is released? Which should hopefully be in the very near future.

**Comment by cgewecke** [comment]
@njgheorghita Yes sounds good!

**Comment by njgheorghita** [comment]
Hijacking this issue to update that we should also have a distribution via apt for linux users.
